opentracing/utils.go

```diff
@@ -0,0 +1,27 @@
+package opentracing
+
+import (
+	"encoding/base64"
+	"fmt"
+	"net/http"
+)
+
+const (
+	OpenTracingContextIDHeader = "OpenTracing-Context-Id"
+)
+
+func AddContextIDToHttpHeader(ctxID ContextID, h http.Header) {
```

Comment:
I'm not sure how many such util functions we would want to bake into the library, but I wanted to write up a few as examples.